### ISSUE TITLE
Server Caching

### ISSUE THREAD

**Issue by programmersharp** [pleerock/routing-controllers#4]
Is it possible to cache action results on the server side?

In ASP Web API it can be achieved the way below (though through external lib):
`[CacheOutput(ClientTimeSpan = 100, ServerTimeSpan = 100)]
public string Get()
{
    return "s";
}`

**Comment by pleerock** [comment]
`through external lib` which lib? if caching is a part and responsibility of that lib, then that lib should provide a decorators for the caching

**Comment by programmersharp** [comment]
Let me rephrase my question. Is it possible to cache controller results on the server side in an elegant way?

**Comment by pleerock** [comment]
yes since this library is using express.js there are libraries like https://www.npmjs.com/package/apicache https://www.npmjs.com/package/express-redis-cache https://www.npmjs.com/package/route-cache and others you need to check.
